youtube_transcript_api/_api.py

```diff
@@ -61,18 +61,21 @@ def list_transcripts(cls, video_id, proxies=None, cookies=None):
         :type proxies: {'http': str, 'https': str} - http://docs.python-requests.org/en/master/user/advanced/#proxies
         :param cookies: a string of the path to a text file containing youtube authorization cookies
         :type cookies: str
+        :param verify: the path to a CA_BUNDLE file or directory with certificates of trusted CAs
+        :type verify: str
         :return: the list of available transcripts
         :rtype TranscriptList:
         """
         with requests.Session() as http_client:
             if cookies:
                 http_client.cookies = cls._load_cookies(cookies, video_id)
             http_client.proxies = proxies if proxies else {}
+            http_client.verify = verify if verify else {}
```

Comment:
`verify` can also be a boolean so this condition can have unintended side-effects when `verify` is set to `False`. Also, it should not be set to `{}` in any case under any circumstances.
Just do this instead
```python
if verify is not None:
    http_client.verify = verify
```